spec/features/schemes_spec.rb

```diff
@@ -149,20 +149,45 @@
         context "when I click to see individual scheme" do
           let(:scheme) { schemes.first }
 
-          before do
-            click_link(scheme.service_name)
+          context "when there is a secondary client group" do
+            before do
```

Comment:
I can see that in the context where you expect it not to have a secondary scheme that you explicitly set it not to, for me I would expect this context you've created to be responsible for making sure that it is the case. 
Looking at the file as a whole though it's one of them things that someone will fix after they break your test by changing something in a wider scope so I guess it's debatable either way

Reply:
I take your point but think this is fine - the only reason we explicitly test the "secondary client group" text doesn't appear in the other test if that there is no client group name for us to test non-existence of in that case, whereas here the secondary_client_group name appearing is enough to see that the correct details are being displayed